analysis/notebooks/README.md

```diff
@@ -1,9 +1,6 @@
-# Example Jupyter / Google Colab Notebooks
+# Example Google Colab / Jupyter Notebooks
 
+- [![Open example.ipynb In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/fuzzbench/blob/master/analysis/notebooks/example.ipynb) A basic example of using analysis functions whose results are not included in the default report.
 
-## Ranking
+- [![Open ranking.ipynb In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/fuzzbench/blob/master/analysis/notebooks/ranking.ipynb) A simple example that compares the existing ranking methods available in the [data_utils](../data_utils.py) module.
```

Comment:
Maybe we should go further and include a link to a notebook for the experiment in the reports?

Reply:
Yes! Let me do that in a following pr.